### PR TITLE
Fix email templates and charts page

### PR DESCRIPTION
**Pull request by stilet00**

### CODE REVIEW

File: src/api/controllers/chartsController.js [hunk 1]
```diff
@@ -209,7 +209,7 @@ var fetchRawBalanceDays = function (year) {
 							{
 								$project: {
 									year: "$_id.year",
-									month: { $toString: "$_id.month" },
+									month: { $toInt: "$_id.month" },
```
Comment (remind1990) [inline src/api/controllers/chartsController.js:212]: if you want to calc anything in aggrigation use  $toDouble than convert result back to string if you want result to be as string

File: src/api/controllers/chartsController.ts [hunk 2]
```diff
@@ -68,7 +68,7 @@ const fetchRawBalanceDays = async (year: string): Promise<RawMonthData[]> => {
 		{
 			$project: {
 				year: "$_id.year",
-				month: { $toString: "$_id.month" },
+				month: { $toInt: "$_id.month" },
```
Comment (remind1990) [inline src/api/controllers/chartsController.ts:71]: sa,e

File: src/api/email-api/email-templates/getTranslatorsEmailTemplate.js [hunk 3]
```diff
@@ -173,28 +173,28 @@ const getTranslatorsEmailTemplateHTMLCode = (translatorInfoForEmailLetter) => {
                                                 <tfoot>
                                                     <tr>
                                                         <td class="container__tfoot-td" colspan="3">
-                                                            Yesterday: ${
-																															translatorInfoForEmailLetter.yesterdayTotal
-																														} <img src="cid:dollar-sign" width="16" height="16" alt="dollar" style="vertical-align: sub"></img>
+                                                            Yesterday: ${translatorInfoForEmailLetter.yesterdayTotal?.toFixed(
+																															2,
```
Comment (remind1990) [inline src/api/email-api/email-templates/getTranslatorsEmailTemplate.js:177]: we are using to fixed but are we sure it is gonna be a string?

File: src/api/controllers/chartsController.js (hunk 1 above)
Comment (stilet00) [reply src/api/controllers/chartsController.js:212]: this is used to convert to number so sorting months by month number works properly

File: src/modules/Charts/ChartsContainer.js [hunk 4]
```diff
@@ -63,18 +62,24 @@ function ChartsContainer() {
...
-												? chartsData[index + 1]
-												: null
-										}
-										graph={month}
-										index={index}
-										key={index}
-									/>
-								))}
+								{chartsData.map((mothData, index) => {
+									const currentMonthNumber = Number.parseInt(
```
Comment (remind1990) [inline src/modules/Charts/ChartsContainer.js:66]: why we use Number.parseInt?

File: src/api/email-api/email-templates/getTranslatorsEmailTemplate.js (hunk 3 above)
Comment (stilet00) [reply src/api/email-api/email-templates/getTranslatorsEmailTemplate.js:177]: .toFixed is method of Number type

File: src/modules/Charts/ChartsContainer.js (hunk 4 above)
Comment (stilet00) [reply src/modules/Charts/ChartsContainer.js:66]: Number() is both more flexible and strict. Number can interpret a value in scientific notation (“7e2” = 700), but will NaN anything that isnt actually a number format (“80px” = NaN), whereas parseInt just reads characters until it finds a non-digit and stops.

**Review by remind1990 — APPROVED** [review]

**Pull request merged**